### DISCUSSION TITLE
Two-factor authentication(2FA) can't use china phone number

### DISCUSSION THREAD

**Discussion by ningmingxiao** [community/community#54401] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Two-factor authentication(2FA) can't use china phone number

**Comment by sabrysm** [comment]
It is possible that some online services or websites do not support phone numbers from certain countries, including China, for two-factor authentication (2FA) purposes. This may be due to various reasons, such as technical limitations or security concerns.

If you are trying to set up 2FA and your China phone number is not supported, you may want to consider using an alternative method, such as a different phone number or an authentication app like Google Authenticator or Authy. Some websites and services also offer backup codes that you can use in case you do not have access to your phone or authentication app.

Alternatively, you can contact the customer support of the website or service you are trying to use and inquire about their supported phone number regions and possible alternatives for 2FA.

**Reply by zion-chuu** [reply]
It's impossible to say it's not supported, Google account can be verified by phone in China.
China does not have phone verification, but Taiwan does, I feel the deep malice.
GitHub will soon pay a heavy price for what it has done, just wait and see

**Reply by lingfeng01** [reply]
Why is 2FA SMS verification not available in China? Google can verify through SMS

**Reply by Yahqubola** [reply]
I will soon get along
Obviously I speculate

**Comment by Yahqubola** [comment]
Please, give me time to orgañise myself.
